benchmarks/benches/room_bench.rs

```diff
@@ -61,7 +61,7 @@ pub fn receive_all_members_benchmark(c: &mut Criterion) {
     );
 
     runtime
-        .block_on(base_client.set_session_meta(
+        .block_on(base_client.set_or_reload_session(
```

Comment:
I'm probably late to the party, but let me add my suggestion anyways since `set_or_reload()` seems to imply we're doing one or the other.

Since the issue here is that there's more to the function than just setting the session metadata, perhaps a more higher level concept could be introduced.

Something that describes in fully that after you call this method the `Client` object will become usable to send/receive messages.

I was thinking that the `Client` might become `active`, so perhaps `activate_client()` and we describe what an active client is:

1. One that has the session meta set (user id, device id, access token)
2. Has loaded cached data from storage

But feel free to ignore this suggestion since this has been going on for quite a bit now.